app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -228,24 +235,38 @@ const SurveyObservations = () => {
       type: ['Out of Range'],
       label: 'Values are out of range'
     },
+    formatting_errors: {
+      type: ['Unexpected Format'],
+      label: 'Unexpected formats in the values provided'
+    },
     miscellaneous: { type: ['Miscellaneous'], label: 'Miscellaneous errors exist in your file' }
   };
 
-  type SubmissionMessages = { [key: string]: string[] };
+  type SubmissionErrors = { [key: string]: string[] };
+  type SubmissionWarnings = { [key: string]: string[] };
 
-  const submissionMessages: SubmissionMessages = {};
+  const submissionErrors: SubmissionErrors = {};
+  const submissionWarnings: SubmissionWarnings = {};
 
   const messageList = submissionStatus?.messages;
 
   if (messageList) {
     Object.entries(messageGrouping).forEach(([key, value]) => {
       messageList.forEach((message) => {
-        if (value.type.includes(message.type)) {
-          if (!submissionMessages[key]) {
-            submissionMessages[key] = [];
+        if (value.type.includes(message.type) && message.class === ClassGrouping.ERROR) {
```

Comment:
nit: can you combine the 2 if statements (for the value.type.includes portion)

Reply:
Oh right .. forgot